tests/StrictObjects/SuggesterTest.phpt

```diff
@@ -29,9 +31,9 @@ class SuggesterTest extends \Tester\TestCase
 	 * 7       2                2
 	 * 8       3                2
 	 *
-	 * @return mixed[]
+	 * @return mixed[][]|NULL[][]
```

Comment:
In other annotations we're using lowercase null. Same problem applies to several more methods below.